products/appengine/terraform.yaml

```diff
@@ -107,7 +107,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "web_service"
         vars:
           bucket_name: "appengine-test-bucket"
-
+  ServiceSplitTraffic: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "{{service_id}}"
+    import_format: ["apps/{{project}}/services/{{service_id}}"]
+    mutex: "apps/{{project}}/services/{{service_id}}"
```

Comment:
Most AppEngine operations need a mutex at the project level, `services/{{service_id}}` can be removed.